proxysql_binlog_reader.cpp

```diff
@@ -223,9 +223,9 @@ bool daemonize_phase3() {
 	return true;
 }
 
-void daemonize_phase1(char *argv0) {
+void daemonize_phase1(char *argv0, std::string pid_file) {
 	int rc;
-	daemon_pid_file_ident="/tmp/proxysql_mysqlbinlog.pid";
+	const char *daemon_pid_file_ident=pid_file.c_str();
 	daemon_log_ident=daemon_ident_from_argv0(argv0);
 	rc=chdir("/tmp");
```

Comment:
this is also dubious, might be good to either allow passing in a working directory in addition or instead of the pid